assets/dev/js/frontend/handlers/base-tabs-v2.js

```diff
@@ -0,0 +1,223 @@
+export default class BaseTabsV2 extends elementorModules.frontend.handlers.Base {
```

Comment:
Add the updated a11y to here

Reply:
Cannot tabs-v2.scss does not exist here but in infra.